packages/react-scripts/config/webpack.config.js

```diff
@@ -139,6 +139,11 @@ module.exports = function (webpackEnv) {
 
   const shouldUseReactRefresh = env.raw.FAST_REFRESH;
 
+  // SHOULD_USE_CACHE_LOADER - this environment variable is not from Create React App side, it shouldn't be added to `env.js` file to get its value, and it should be deleted when cache strategy is ready for all the web teams at Skyscanner
+  // Warning: cache-loader won't be needed when webpack 5 is there
+  const shouldUseCacheLoader =
+    isEnvProduction && process.env.SHOULD_USE_CACHE_LOADER;
```

Comment:
Should this not be `isEnvDevelopment`? As CI doesn't use the variable production right? So would be still a development environment as seen below
https://github.com/Skyscanner/backpack-react-scripts/pull/99/files#diff-8e25c4f6f592c1fcfc38f0d43d62cbd68399f44f494c1b60f0cf9ccd7344d697L158

Reply:
Since few developers run `build` command locally, it is OK for using `cache-loader` only in production. But Keeping it as before will also make sense. will remove `isEnvProduction` flag and just use `SHOULD_USE_CACHE_LOADER` to control.